kll/test/kll_sketch_deserialize_from_java_test.cpp

```diff
@@ -75,4 +75,29 @@ TEST_CASE("kll double", "[serde_compat]") {
   }
 }
 
+// numbers are padded with leading spaces so that natural order works
```

Comment:
this is automatic in the serde? there doesn't seem to be anything in the code that'd handle that?

Reply:
this is how it was done on the Java side during sketch updates presumably to avoid supplying a custom comparator